app/screens/search/search.js

```diff
@@ -440,10 +443,11 @@ class Search extends Component {
             this.props.navigator.dismissModal({animationType: 'none'});
 
             markChannelAsRead(channelId, currentChannelId);
-            setChannelLoading();
             viewChannel(channelId, currentChannelId);
             setChannelDisplayName(displayName);
-            handleSelectChannel(channelId);
+            InteractionManager.runAfterInteractions(() => {
```

Comment:
If we've fixed a lot of the re-rendering, is it possible to call handleSelectChannel without the runAfterInteractions? That can be left for 1.4 to investigate though if we think it might be possible

Reply:
perhaps if we end up using the native animations for a new drawer we can get rid of this, if not this needs to stay

Reply:
I'm looking into using native animations with the drawer today to see if we can get rid of that slight pause before everything starts to load.